app/forms/flood_risk_engine/steps/base_address_form.rb

```diff
@@ -40,32 +40,40 @@ def assign_to_enrollment(_address)
 
       protected
 
-      attr_reader :address_to_assign
+      attr_reader :assignable_address
 
-      def assign_via_uprn
+      def find_and_validate_address_via_uprn
         # Use UPRN to find address data
         address_attributes = find_by_uprn
 
         if address_attributes
-          validator = AddressValidator.new(address_attributes)
+          # Given that the address data comes from service we should not validate it here, as these addresses
+          # are already displayed to the user for selection so then raising an error is goign to be very confusing
+          # N.B Addresses from the service may not have Street address but always have UPRN and Postcode
 
-          if validator.validate
-            @address_to_assign = Address.new(address_attributes)
-            logger.debug("LocalAuthority address now #{@address_to_assign.inspect}")
-          else
-            # not too sure how to merge this validation into Form automatically so for now
-            # manually clone error messages
-            logger.debug("Address Valid #{validator.valid?} Errors [#{validator.errors.inspect}]")
+          # Validation code left for reference, TODO: can be deleted next Sprint
```

Comment:
The problem is, it won't be. If code is no longer needed it needs to be removed. Otherwise the code base just gets filled with obsolete and/or commented out code. We have source control, and can step back via that to get to old code patterns.